src/com/github/toxuin/griswold/Interactor.java

```diff
@@ -47,31 +47,31 @@ private void loadDefaultItems() {
         repairableTools.add(Material.IRON_PICKAXE);
         repairableTools.add(Material.IRON_SWORD);
         repairableTools.add(Material.IRON_HOE);
-        repairableTools.add(Material.IRON_SPADE);              // IRON TOOLS
+        repairableTools.add(Material.IRON_SHOVEL);              // IRON TOOLS
 
-        repairableTools.add(Material.WOOD_AXE);
-        repairableTools.add(Material.WOOD_PICKAXE);
-        repairableTools.add(Material.WOOD_SWORD);
-        repairableTools.add(Material.WOOD_HOE);
-        repairableTools.add(Material.WOOD_SPADE);              // WOODEN TOOLS
+        repairableTools.add(Material.WOODEN_AXE);
```

Comment:
@toxuin we either try to support back compatibility here or ignore it. Either way, the user needs to change the config, since the only way to get legacy item IDs is via appending "LEGACY_" prefix.

Reply:
I think there should be a warning somewhere that since 1.13/1.14 the item config needs to be revised and we're should be good.

It would not hurt to add a warning when a user has garbage (or legacy stuff that doesn't exist anymore) in their config.